shell/platform/embedder/tests/embedder_test_context.h

```diff
@@ -121,6 +136,8 @@ class EmbedderTestContext {
   size_t software_surface_present_count_ = 0;
   std::mutex gl_get_fbo_callback_mutex_;
   GLGetFBOCallback gl_get_fbo_callback_;
+  std::mutex gl_present_callback_mutex_;
```

Comment:
I suppose you could use a common mutex for these callbacks now.

Reply:
Done!